gfx/UltimateCataclysm/pngs_normal_32x32/vehicle/scooter/scooter_wheels.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "id": [ "vp_wheel_small_scooter", "vp_wheel_small_scooter_steerable" ],
+    "fg": [ "vp_wheel_scooter_rotN", "vp_wheel_scooter_rotW", "vp_wheel_scooter_rotS", "vp_wheel_scooter_rotE" ],
+    "rotates": true,
+    "bg": ""
+  },
+  {
+    "id": [ "vp_wheel_small_scooter_rear" ],
+    "fg": [ "vp_wheel_scooter_rear_rotN", "vp_wheel_scooter_rear_rotW", "vp_wheel_scooter_rear_rotS", "vp_wheel_scooter_rear_rotE" ],
+    "rotates": true,
+    "bg": ""
```

Comment:
Both `    "rotates": true` and `"bg": ""` are redundant here. `compose.py` was edited to not panic on missing `bg` and array of 4 values sets `rotates` to `true` anyway. At least it should.